File: 2600-k-items-with-the-maximum-sum/2600-k-items-with-the-maximum-sum.java
public class Solution {

    public  int kItemsWithMaximumSum(int numOnes, int numZeros, int numNegOnes, int k) {
        ArrayList<Integer> bagItems = new ArrayList();
        for (int i = 0; i < numOnes; i++) {
            bagItems.add(1);
        }
        for (int i = 0; i < numZeros; i++) {
            bagItems.add(0);
        }
        for (int i = 0; i < numNegOnes; i++) {
            bagItems.add(-1);
        }
        
        Collections.sort(bagItems, Collections.reverseOrder());
        int sumOfKItems = 0;
        for (int i = 0; i < k; i++) {
            sumOfKItems += bagItems.get(i);
        }

        return sumOfKItems;
    }
}

File: 2600-k-items-with-the-maximum-sum/2600-k-items-with-the-maximum-sum.py
class Solution:
    def kItemsWithMaximumSum(self, numOnes: int, numZeros: int, numNegOnes: int, k: int) -> int:
        bagItems=[]
        sumOfItems=0
        for naol in range(numOnes):
            bagItems.append(1)
        for naol in range(numZeros):
            bagItems.append(0)
        for naol in range(numNegOnes):
            bagItems.append(-1)
        bagItems.sort(reverse=True)
        for naol in range(k):
            sumOfItems+=bagItems[naol]
        return sumOfItems
        